main.js

```diff
@@ -81,23 +104,49 @@ function updateLatestGuess() {
   resetButton.disabled = true;
 }
 
+function updateRange() {
+  minSpan.innerText = minRange.value;
+  maxSpan.innerText = maxRange.value;
+  var min = parseInt(minRange.value);
+  var max = parseInt(maxRange.value);
+  correctNumber = getRandomRange(min, max);
+  minRange.value = "";
+  maxRange.value = "";
+  updateButton.disabled = true;
+}
+
 function checkGuess(guessInput, feedbackMessage) {
   var currentGuess = parseInt(guessInput.value);
+  var tooHigh = "that's too high";
+  var tooLow = "that's too low";
+  var perfectGuess = "BOOM!";
+
   if (currentGuess === correctNumber) {
-    feedbackMessage.innerText = "BOOM!";
+    feedbackMessage.innerText = perfectGuess;
   } else if (currentGuess > correctNumber) {
-    feedbackMessage.innerText = "that's too high";
+    feedbackMessage.innerText = tooHigh;
   } else if (currentGuess < correctNumber) {
-    feedbackMessage.innerText = "that's too low";
+    feedbackMessage.innerText = tooLow;
   } else {
     feedbackMessage.innerText = "something went wrong";
   }
 }
 
-function updateRange() {
-  minSpan.innerText = minRange.value;
-  maxSpan.innerText = maxRange.value;
+function checkRange() {
   var min = parseInt(minRange.value);
   var max = parseInt(maxRange.value);
-  correctNumber = getRandomRange(min, max);
+  if (min >= max) {
+    errorIcon.style.visibility = 'visible';
+    maxRange.style.borderColor = '#dd1972';
+  } else {
+    errorIcon.style.visibility = 'hidden';
+    maxRange.style.border = '1px solid #d0d2d3';
+  }
+  if (min < max) {
+    updateButton.disabled = false;
+    updateButton.style.backgroundColor = '#6e6e6e';
+  } else {
+    updateButton.disabled = true;
+    updateButton.style.backgroundColor = '#d0d2d3';
+  }
```

Comment:
You can  combine the two if else statements into one, since they're basically saying the same thing but in reverse.  If the maximum is smaller than or equal to the min then the button will be disabled, and if it is greater than the min the button will be enabled.  Also you don't need to style the button in JS, the :disabled class in CSS will do that for us.

`if (min >= max) {`
    `errorIcon.style.visibility = 'visible';`
    `maxRange.style.borderColor = '#dd1972';`
    `updateButton.disabled = true;`
    `updateButton.style.backgroundColor = '#d0d2d3';`
 ` } else {`
    `errorIcon.style.visibility = 'hidden';`
    `maxRange.style.border = '1px solid #d0d2d3';`
    `updateButton.disabled = false;`
    `updateButton.style.backgroundColor = '#6e6e6e';`
`  }`